plugins/module_utils/network/sonic/argspec/facts/facts.py

```diff
@@ -59,7 +59,8 @@ def __init__(self, **kwargs):
         'bfd',
         'copp',
         'route_maps',
-        'stp'
+        'stp',
+        'sFlow'
```

Comment:
Everything in this list is lower-case; let's keep it consistent.

Reply:
does this apply to the file names as well?

Reply:
oh, yeah I didn't even notice that. Let's make all the files lower-case as well.

Reply:
addressed this in next commit